### PR TITLE
#26 Add example mapping from Map to property

### PR DESCRIPTION
**Pull request by xenteros**
*Added Source class with single field: Map<String, Object>,
*Added Target class with two fields, String ip and String server,
*Added MappingUtil with two interfaces for each of the properties,
*Added SourceTargetMapper,
*Covered with positive and negative tests.
*Updated README file

### CODE REVIEW

File: mapstruct-mapping-from-map/pom.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.bsc.maven</groupId>
+                <artifactId>maven-processor-plugin</artifactId>
```
Comment (filiphr) [inline mapstruct-mapping-from-map/pom.xml]: Can you please use the `maven-compiler-plugin` instead? We are trying to use that one in the examples as it is the best way to do it with maven.

**Comment by xenteros** [comment]
@filiphr I have finally found some time to apply requested changes. Hopefully it's good now :)

**Comment by filiphr** [comment]
@xenteros it looks better now. I'll merge it later today. One question. can I merge the 2 commits into one? I see that the authors of the commits are different.

**Comment by xenteros** [comment]
@filiphr yes of course, could you pick author @xenteros? The other was the reason of not changing the profile after switching from work to pleasure :)

**Comment by filiphr** [comment]
This has been merged in 38bb29a.  Thanks a lot for the example @xenteros.

Just as info, I added some small things in 937c2fc.

**Pull request closed without merging**